person.rb

```diff
@@ -26,8 +27,8 @@ def can_use_services
   def correct_name
     @name
   end
-end
 
-person = Person.new(1, 19, 'Zunair')
-puts(person.send(:of_age?))
-puts(person.can_use_services)
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Let's modify this method as we did for the Book class. The only change here is, we have access to the Person instance and we need to pass a date and a Book. ✍️

```
def add_rental(book, date)
  Rental.new(date, book, self)
end
```

Reply:
Thanks! the change has been made